src/main/java/com/robertx22/age_of_exile/saveclasses/item_classes/GearTooltipUtils.java

```diff
@@ -183,12 +189,14 @@ public static void BuildTooltip(GearItemData gear, ItemStack stack, List<Compone
         }
 
 
-        if (Screen.hasShiftDown() == false) {
```

Comment:
This is a bit of a nitpick but please don't format my code 

== false or ! both fine and I usually use == false when I want to make sure I don't accidentally not see the tiny ! symbol.

You don't have to fix this one

Reply:
I don't even know why I do this, probably just misclick or smth.